app/views/reports/regions/_index_tree.html.erb

```diff
@@ -0,0 +1,75 @@
+<div id="facility-access" data-page="show">
+
+  <% regions.each do |org, regions_by_district| %>
+
+  <div class="card access-tree show">
+    <div class="access-tree__section-header">
+      <%= org.name %>
+    </div>
+
+    <ul>
+      <% regions_by_district.each do |district, regions_by_block| %>
+      <li>
+        <div class="access-item">
+
+          <div class="access-item__dropdown">
+            <i class="fas fa-caret-down light"></i>
+          </div>
+
+          <div class="form-check show">
+            <%= link_to_if(accessible_district?(district), district.name, reports_region_path(district, report_scope: district.region_type)) %>
+          </div>
+
+          <div class="access-item__padding"></div>
+        </div>
+
+        <ul class="access-item-wrapper block">
+          <% sorted_blocks = regions_by_block.sort_by { |block, regions| block.name } %>
+          <% sorted_blocks.each do |block, regions| %>
+            <li>
+              <div class="access-item block collasped">
+                <span class="spacer"></span>
+
+                <div class="access-item__dropdown">
+                  <i class="fas fa-caret-down light"></i>
+                </div>
+
+                <div class="form-check show">
+                  <%= link_to_if(accessible_block?(block), block.name, reports_region_path(block, report_scope: block.region_type)) %>
+                </div>
+
+                <div class="access-item__padding"></div>
+              </div>
+
+            <ul class="access-item-wrapper block">
+            <% regions.sort_by(&:name).each do |region| %>
+              <li>
+                <div class="access-item <%= region.region_type %> collasped ">
+                  <span class="spacer"></span><span class="spacer"></span><span class="spacer"></span>
+
+                  <div class="form-check show">
+                    <%= link_to region.name, reports_region_path(region, report_scope: region.region_type) %>
```

Comment:
If we've gotten this far, we know the facility is one the user can view reports on.